spec/ruby/language/predefined/toplevel_binding_spec.rb

```diff
@@ -0,0 +1,24 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+describe "The TOPLEVEL_BINDING constant" do
+  before :each do
```

Comment:
@eregon if you change this to `before :all`, I think you get errors in our logic for detecting slow specs

```
1)
An exception occurred during: before :all ERROR
NoMethodError: undefined method `mspec_old_ruby_exe' for #<MSpecEnv:0x121a>:MSpecEnv
/Users/chrisseaton/Documents/truffleruby/truffleruby/spec/truffle.mspec:216:in `method_missing'
/Users/chrisseaton/Documents/truffleruby/truffleruby/spec/truffle.mspec:216:in `send'
/Users/chrisseaton/Documents/truffleruby/truffleruby/spec/truffle.mspec:216:in `ruby_exe'
/Users/chrisseaton/Documents/truffleruby/truffleruby/spec/ruby/language/predefined/toplevel_binding_spec.rb:5:in `block (2 levels) in <top (required)>'
/Users/chrisseaton/Documents/truffleruby/truffleruby/src/main/ruby/core/enumerable.rb:478:in `block in all?'
/Users/chrisseaton/Documents/truffleruby/truffleruby/src/main/ruby/core/enumerable.rb:478:in `each'
/Users/chrisseaton/Documents/truffleruby/truffleruby/src/main/ruby/core/enumerable.rb:478:in `all?'
/Users/chrisseaton/Documents/truffleruby/truffleruby/spec/ruby/language/predefined/toplevel_binding_spec.rb:3:in `<top (required)>'
/Users/chrisseaton/Documents/truffleruby/truffleruby/src/main/ruby/core/kernel.rb:333:in `load'
/Users/chrisseaton/Documents/truffleruby/truffleruby/src/main/ruby/core/kernel.rb:333:in `load'
```

Reply:
Is `ruby_exe` needed here or would `require` be enough?

Reply:
It seems the main script file influences TOPLEVEL_BINDING so indeed we need `ruby_exe` then.

Reply:
Yes these specs are all about top-level things, so I think it makes sense to isolate them.